### ISSUE TITLE
You have no authorized access to projects When overwrite framework.properties

### ISSUE THREAD

**Issue by dbwodlf3** [rundeck/rundeck#9291]
![image](https://github.com/user-attachments/assets/9ffafbb5-e8c3-45f9-baef-0bc79dfc506c)

```
#framework.properties.toml
[[template]]
src = "${REMCO_TEMPLATE_PATH}/framework.properties"
dst = "${RUNDECK_DATA}/etc/framework.properties"
mode = "0644"
```

```
#framework.properties
# framework.properties -
#

# ----------------------------------------------------------------
# Server connection information
# ----------------------------------------------------------------

framework.server.name = localhost
framework.server.hostname = localhost
framework.server.port = 4440
framework.server.url = http://localhost:4440

# ----------------------------------------------------------------
# Installation locations
# ----------------------------------------------------------------

rdeck.base=/workspace/data

framework.projects.dir=/workspace/data/projects
framework.etc.dir=/workspace/data/etc
framework.var.dir=/workspace/data/var
framework.tmp.dir=/workspace/data/var/tmp
framework.logs.dir=/workspace/data/var/logs
framework.libext.dir=/workspace/data/libext

# ----------------------------------------------------------------
# SSH defaults for node executor and file copier
# ----------------------------------------------------------------

framework.ssh.keypath = /root/.ssh/id_rsa
framework.ssh.user = root

# ssh connection timeout after a specified number of milliseconds.
# "0" value means wait forever.
framework.ssh.timeout = 0

# ----------------------------------------------------------------
# Auto generated server UUID: 254e9c5b-bf42-4bf6-add4-5bf4e79bbe48
# ----------------------------------------------------------------
rundeck.server.uuid = 254e9c5b-bf42-4bf6-add4-5bf4e79bbe48

# ----------------------------------------------------------------
# System-wide global variables.
# ----------------------------------------------------------------

# Expands to ${globals.var1}
#framework.globals.var1 = value1

# Expands to ${globals.var2}
#framework.globals.var2 = value2


# ----------------------------------------------------------------
# API Related settings
# ----------------------------------------------------------------
rundeck.tokens.file=/workspace/data/etc/tokens.properties

```

In a Docker container, I'm trying to write an initialization script for Rundeck. However, after applying the settings above, I encounter the message "You have no authorized access to projects." when logging in as the admin account. I suspect that Rundeck relies on the rundeck.server.uuid for certain operations, and the issue might be due to a mismatch between the original UUID and the one that has been set. How can I resolve this?

**Comment by Daryes** [comment]
Do a test without the rundeck.server.uuid parameter, it is required for a cluster or specific license cases.
Just to validate.
For the record, I'm currently running Rundeck with this removed, and without any problem.

This said, check the logs, you might have the cause here.
One suspect is your `framework.ssh.keypath = /root/.ssh/id_rsa` parameter : the docker image switch internally to the `rundeck` user, so it won't be able to get the ssh private key under /root/

**Comment by dbwodlf3** [comment]
Thank you for your kind response.

After trying it out, I found that the issue was different from what you mentioned. When Rundeck runs for the first time, it initializes the framework.properties and ACL files under the etc folder. In the code I wrote, I run remco first and then start Rundeck. During that step, the framework.properties file is created under the etc folder of Rundeck before it has been fully initialized.

I believe Rundeck skips several initialization tasks, including the creation of ACL files, if the framework.properties file already exists. As a result, the issue in the screenshot occurred because the ACL files were not generated.

For now, I have worked around the issue by allowing Rundeck to complete its initialization process before proceeding with other tasks.
